app/views/user.py

```diff
@@ -491,14 +491,17 @@ def delete(self, user_type_id):
                         UserTypeSchema
         """
         try:
-            user_type = UserType.query.filter(UserType.id == user_type_id).first()
-            db.session.delete(user_type)
+            user_type_data = UserType.query.filter(UserType.id == user_type_id).first()
+            if not UserTypeObjectNotFound:
+                raise UserTypeObjectNotFound
```

Comment:
Pass message from here and use it in the response and while printing the logs.

Reply:
Made the changes as suggested.